packages/shared/src/utils/slots.ts

```diff
@@ -28,3 +28,45 @@ export const getSlotTimespan = (
   // return time string
   return `${startTime} - ${endTime}`;
 };
+
+// #region CalculateInterval
+export const calculateIntervalDurationInMinutes = (interval: string | null) => {
+  if (interval === null) {
+    return 0;
+  }
```

Comment:
Finally 👌 
I've been meaning to move this here forever now, but always seem to forget, tnx